Test/Applications/test_xia2setup.py

```diff
@@ -0,0 +1,41 @@
+import mock
+import os
+import sys
+from importlib import reload
+
+from xia2.Handlers.XInfo import XInfo
+
+
+def test_write_xinfo_split_sweep(dials_data, tmp_path, monkeypatch):
+    # This test partially exercises the fix to xia2/xia2#498
+    # run in tmp dir so droppings not cluttering source area
+    monkeypatch.chdir(tmp_path)
```

Comment:
Why not use the `run_in_tmpdir` fixture that we use everywhere else?

Reply:
For some reason I thought that was about running the things you run in a tmpdir, rather than running the code in there, and google gave me this monkeypatch solution which worked 😉

Respectfully acknowledge that a more standard solution is better 👍

Reply:
The `run_in_tmpdir` fixture literally is just a wrapper around the `tmpdir` fixture that does a `tmpdir.chdir()` before running your test:
https://github.com/dials/dials/blob/9fbf9fcc2dafcb03a916cb7a83bd875e2584004c/conftest.py#L79-L84

Reply:
I don't like `run_in_tmpdir` but acknowledge it is useful because xia2 has so many implicit file operations :disappointed: 

The reason I don't like it is because any `chdir` operation changes the working directory, which is essentially a per-process singleton. I know we don't run any parallel threads here (`pytest -n` runs in processes), but still it feels unclean.

I made some headway towards what I believe is overall a better approach in #424 where I added a `.path` attribute (type pathlib.Path) to `XProject`, which now no longer depends on a working directory.